pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -23,12 +23,7 @@
 /** Options supported by {@link FhirEtl}. */
```

Comment:
Yeah, these flags in general have become a mess and we need to do more clean-up (#361) but that is beyond this PR. In fact `isJdbcModeFhir` is not a good name because that is really `isJdbcOpenmrs`; and then the question is how much of the OpenMRS specific functionality we want to keep. So let's leave this for when we fix #361.